app/Services/Manager.php

```diff
@@ -528,4 +528,26 @@ public function sendEmail($user, $contest, $params)
             return null;
         }
     }
+
+    /**
+     * Search for users in Northstar.
+     *
+     * @param  string $query
+     * @return collection|null
+     */
+    public function search($query, $page = null)
+    {
+        // Attempt to fetch all users.
+        $users = $this->northstar->getAllUsers([
+            'search' => [
+                '_id' => $query,
+                'drupal_id' => $query,
+                'email' => $query,
+                'mobile' => $query,
```

Comment:
I'm surprised Northstar doesn't freak out if you supply an email address for a mobile number?